src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -88,13 +88,42 @@ static ucs_status_t uct_gdr_copy_iface_query(uct_iface_h tl_iface,
 
     iface_attr->latency                 = ucs_linear_func_make(1e-6, 0);
     iface_attr->bandwidth.dedicated     = 0;
-    iface_attr->bandwidth.shared        = 6911.0 * UCS_MBYTE;
-    iface_attr->overhead                = 0;
+    iface_attr->bandwidth.shared        = UCT_GDR_COPY_IFACE_DEFAULT_BANDWIDTH;
+    iface_attr->overhead                = UCT_GDR_COPY_IFACE_OVERHEAD;
     iface_attr->priority                = 0;
 
     return UCS_OK;
 }
 
+static ucs_status_t
+uct_gdr_copy_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
+{
+    if (perf_attr->field_mask & UCT_PERF_ATTR_FIELD_BANDWIDTH) {
```

Comment:
As discussed with @yosefe , will add a generic macro for get/set param in a separate PR.
Anyway, it might not be suitable here at all - thus it is not a blocker.